### ISSUE TITLE
Account appears to have been locked by Keybase staff

### ISSUE THREAD

**Issue by bitcoinmandem** [keybase/client#25528]
Hi, my keybase account @bitcoinmandem has suddenly been inaccessible for around 10 days now. I've tried logging in on all of my devices and using my paper key as well but every attempt to log back in has failed. I have been using this application a great deal over the past couple years, and if this is not resolved I will be losing many extremely important conversations and contacts. Please fix this issue, or at least let me know why this has happened.

**Comment by RincewindTVD** [comment]
Are you getting a 202 error 'bad session' and asking if your device is revoke?

If so, I am getting the same.

**Comment by bitcoinmandem** [comment]
Yep, error 202 bad session on all devices, even when using the paper key - interestingly, I still receive an email that the paper key has been used whenever I attempt to log in
